test_regress/t/t_stream_integer_type_debug.pl

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2003 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
+compile(
+#    fails => $Self->{vlt_all},
```

Comment:
```suggestion
```

Reply:
test_regress/t/t_stream_integer_type_debug.pl is JUST to demonstrate the basic byte pack concept. I have no plan to release it.